pkg/staticnetworkconfig/generator.go

```diff
@@ -186,7 +187,12 @@ func (s *StaticNetworkConfigGenerator) validateInterfaceNamesExistence(macInterf
 		switch interfaceType.String() {
 		case "802-3-ethernet", "ethernet":
 			if !lo.Contains(interfaceNames, interfaceName.String()) {
-				return errors.Errorf("mac-interface mapping for interface %s is missing", interfaceName.String())
+				physInt := regexp.MustCompile("^en[PsvXbucaipod]")
```

Comment:
What if a user selects a logical name that matches the regular expression, but in practice, it's just a randomly chosen logical name? Isn't that a situation that could happen?

Reply:
In that case, yes, the name would pass validation.